src/sciline/_provider.py

```diff
@@ -50,27 +64,30 @@ def __init__(
 
     @classmethod
     def from_function(cls, func: ToProvider) -> Provider:
+        """Construct from a function or other callable."""
         return cls(func=func, arg_spec=ArgSpec.from_function(func), kind='function')
 
     @classmethod
     def parameter(cls, param: Any) -> Provider:
+        """Construct a provider that always returns the given value."""
         return cls(
             func=lambda: param,
             arg_spec=ArgSpec.null(),
             kind='parameter',
             location=ProviderLocation(
-                name=f'param({type(param).__name__})', module='sciline'
+                name=f'param({type(param).__name__})', module=_module_name(param)
             ),
         )
 
     @classmethod
-    def table(cls, param: Any) -> Provider:
+    def table_row(cls, param: Any) -> Provider:
+        """Construct a provider that returns the label for a table row."""
```

Comment:
I think it is not a row that is being returned, just a single cell (i.e., specific row and column).

Reply:
It is a single value. E.g. in https://scipp.github.io/sciline/user-guide/parameter-tables.html#Computing-results-for-series-of-parameters it is a single `int`, i.e., the `RunID`, but not the `Filename`.

Reply:
Yes, i.e., rename?